core/templates/components/forms/custom-forms-directives/edit-thumbnail-modal.controller.spec.ts

```diff
@@ -33,6 +37,9 @@ describe('Edit Thumbnail Modal Controller', function() {
   const tempBgColor = '';
   const uploadedImage = new File([], 'uploaded.png');
   const uploadedImageMimeType = 'image/svg+xml';
+  importAllAngularServices();
+
+  importAllAngularServices();
```

Comment:
`importAllAngularServices` has been called twice.

Reply:
This is fixed.